infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -104,12 +106,14 @@ private String generateNodeStartCommand(String instanceId, String rmUrlToUse, St
 
         String jythonPath = nsConfig.getString(NSProperties.DEFAULT_JYTHON_PATH);
 
+        String credentialsProperty = " -v " + credentials;
+
         String javaProperties = " -Dproactive.communication.protocol=" + protocol + " -Dpython.path=" + jythonPath +
                                 " -Dproactive.pamr.router.address=" + rmHostname + " -D" + instanceTagNodeProperty +
                                 "=" + instanceId + " " + additionalProperties + " -r " + rmUrlToUse + " -s " + nsName +
                                 nodeNamingOption + " -w " + numberOfNodesPerInstance;
 
-        return "nohup " + javaCommand + javaProperties + "  &";
+        return "nohup " + javaCommand + credentialsProperty + javaProperties + "  &";
```

Comment:
I think it is better to include credentialsProperty in javaProperties otherwise the order of parameters would be cahotic